### PR TITLE
improve GenesisProtocol parameters documentation

### PR DESCRIPTION
**Pull request by dkent600**

### CODE REVIEW

File: lib/wrappers/genesisProtocol.ts [hunk 1]
```diff
@@ -1177,50 +1178,65 @@ export interface GenesisProtocolParams extends TxGeneratingFunctionOptions {
...
    * Default is 1814400 (three weeks).
    */
   preBoostedVotePeriodLimit: number;
   /**
-   * The percentage of the absolute vote that must be exceeded to result in a win.
-   * Must be between 0 and 100.
+   * The percent of the DAO's total supply of reputation that, when exceeded
+   * by the amount of reputation behind a vote (yes or no), will result
+   * in the immediate execution of the proposal, during either the preboosted
+   * or boosted phases. Must be between 0 and 100.
```
Comment (orenyodfat) [inline lib/wrappers/genesisProtocol.ts]: 0<preBoostedVotePeriodLimit <=100

Comment (dkent600) [reply lib/wrappers/genesisProtocol.ts]: @orenyodfat There is currently no validation on preBoostedVotePeriodLimit in Arc.  I'm not sure where I got that statement about the required range of values.

Comment (dkent600) [reply lib/wrappers/genesisProtocol.ts]: @orenyodfat Are you sure you aren't referring to preBoostedVoteRequiredPercentage?

Comment (orenyodfat) [reply lib/wrappers/genesisProtocol.ts]: yea .was referring to preBoostedVoteRequiredPercentage .it is ok. 👍

**Pull request merged**